### PR TITLE
ASA -  PolicySync Cleanup

### PR DESCRIPTION
**Pull request by isaacdurham-okta**
## Description:
- **What's changed?** Updated PolicySync references to reflect GA status.
- **Is this PR related to a Monolith release?** 2023.01.0

Alias URL won't be live until next HOC publish cycle. Maybe Jan 16 or 17?

### Resolves:

* [OKTA-566414](https://oktainc.atlassian.net/browse/OKTA-566414)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ https://app.scaleft.com/v1/teams/${team_name}/servers
 ### Update a Server
 
 <ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/servers/${server_id}" />
-Updates a Server. This endpoint is only available with the PolicySync feature, which is currently in EA.
+Updates a Server. This endpoint is intended for use with the PolicySync feature.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:162]:
```suggestion
Updates a Server. This endpoint is used for the PolicySync feature.
```
Do you want to use the `<ApiLifecycle access="ea" />` if this feature is EA?

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md [hunk 2]
```diff
@@ -2053,7 +2053,7 @@ HTTP 204 No Content
 ### Update a Server on a Project
 
 <ApiOperation method="PUT" url="https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/servers/${server_id}" />
-Updates a Server on a Project. This endpoint is only available with the PolicySync feature, which is currently in EA.
+Updates a Server on a Project. This endpoint is intended for use with the PolicySync feature, see [PolicySync: Attribute-Based Access Control](https://help.okta.com/okta_help.htm?type=asa&id=ext-asa-policysync).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:2056]:
```suggestion
Updates a Server on a Project. This endpoint is used for the PolicySync feature, see [PolicySync: Attribute-Based Access Control](https://help.okta.com/okta_help.htm?type=asa&id=ext-asa-policysync).
```
The help doc alias is not available yet?

**Review by vanngo-okta — APPROVED** [review]
LGTM! Just a few suggestion nits. Approving to not hold you back.

File: packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md (hunk 1 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/reference/api/asa/teams/index.md:162]: It actually left EA already.

File: packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md (hunk 2 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md:2056]: Should be live this week.

**Review by waltergoulet-okta — APPROVED** [review]
LGTM; all mention of PolicySync as an EA feature is gone.

**Pull request merged**